### ISSUE TITLE
What is the batch size of training

### ISSUE THREAD

**Issue by MultiPath** [jychoi118/P2-weighting#2]
Hi, I could not find the details in the original paper about the batch size.
Do you train all 256x256 models on single GPU with batch-size=8? Or you trained them on multi-gpu with larger batch sizes (for example the original DDPM mentioned 64/128).
Thank you very much

**Comment by jychoi118** [comment]
Yes we trained models on a single GPU with batch size 8. It needed about 23GB vram.

**Comment by MultiPath** [comment]
Thanks so much! For AFHQ, how many steps do you need to train the model? Do you use the ema checkpoint?

**Comment by jychoi118** [comment]
For AFHQ, we trained for 300k iteration with dropout (p=0.1). We used ema for all experiments.

**Comment by MultiPath** [comment]
Thanks for the answer!
